src/main/java/net/neoforged/neoforge/registries/datamaps/builtin/AcceptableVillagerDistance.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.registries.datamaps.builtin;
+
+import com.mojang.serialization.Codec;
+import com.mojang.serialization.codecs.RecordCodecBuilder;
+
+/**
+ * Data map value for {@linkplain NeoForgeDataMaps#ACCEPTABLE_VILLAGER_DISTANCES acceptable hostile distances}.
+ *
+ * @param distance the acceptable distance between a villager and a hostile mob.
+ */
+public record AcceptableVillagerDistance(float distance) {
+    public static final Codec<AcceptableVillagerDistance> DISTANCE_CODEC = Codec.FLOAT
+            .xmap(AcceptableVillagerDistance::new, AcceptableVillagerDistance::distance);
+
+    public static final Codec<AcceptableVillagerDistance> CODEC = Codec.withAlternative(
+            RecordCodecBuilder.create(in -> in.group(
+                    Codec.FLOAT.fieldOf("acceptable_hostile_distance").forGetter(AcceptableVillagerDistance::distance)).apply(in, AcceptableVillagerDistance::new)),
```

Comment:
```suggestion
                    Codec.FLOAT.fieldOf("acceptable_villager_distance").forGetter(AcceptableVillagerDistance::distance)).apply(in, AcceptableVillagerDistance::new)),
```
Make sure to regenerate the datamap file after changing this